### ISSUE TITLE
create_standalone_context() fails in moderngl5

### ISSUE THREAD

**Issue by alexgrabner** [cprogrammer1994/ModernGL#173]
I just upgraded from release 4.2.2 to 5.0.4 and the standalone context creation is broken for me. I reverted to 4.2.2, where it works. Thanks for help in advance.

Ubuntu 16.04 64bit
Python 3.6.4

### Input

```
import moderngl
ctx = moderngl.create_standalone_context()
```

### Output

```
   3120     ctx = Context.__new__(Context)
-> 3121     ctx.mglo, ctx.version_code = mgl.create_standalone_context(settings)
   3122     ctx._screen = None
   3123     ctx.fbo = None

Error: cannot choose a visual info
```

**Comment by cprogrammer1994** [comment]
Thank you for the report.

There were 3 implementation for standalone context on linux (X11 EGL OsMesa) unfortunately only X11 supports OpenGL 3.3+ so I restored the original implementation.

```py
python -m pip install "ModernGL>=5.0.5"
```

**Comment by alexgrabner** [comment]
Thanks for the quick feedback, but the issue is still there for me. Any suggestion? Thanks.

X.Org version: 1.18.4
OpenGL version string: 4.6.0 NVIDIA 390.30

# Input
```
import moderngl
print(moderngl.__version__)
ctx = moderngl.create_standalone_context()
```

# Output
```
5.0.5
---------------------------------------------------------------------------
Error                                     Traceback (most recent call last)
...
   3120     ctx = Context.__new__(Context)
-> 3121     ctx.mglo, ctx.version_code = mgl.create_standalone_context(settings)
   3122     ctx._screen = None
   3123     ctx.fbo = None

Error: cannot choose a visual info
```

**Comment by cprogrammer1994** [comment]
This is the only remarkable difference:  https://github.com/cprogrammer1994/ModernGL/commit/f34af6af40a92f90fe790db004f1981bd8f1a97e

please test with the master now:

```py
python -m pip install https://github.com/cprogrammer1994/ModernGL/archive/master.zip
```

The problem does not occure on my computer. Please test it. It will help me a lot.

**Comment by alexgrabner** [comment]
Now it works! Thanks!

**Comment by cprogrammer1994** [comment]
This fix will be part of ModernGL 5.0.6 (will release today)
